src/router_core/modules/edge_addr_tracking/edge_addr_tracking.c

```diff
@@ -31,7 +31,8 @@ struct qdr_addr_endpoint_state_t {
     qdrc_endpoint_t                    *endpoint;
     qdr_connection_t                   *conn;    // The connection associated with the endpoint.
     qdr_addr_tracking_module_context_t *mc;
-    qdr_link_t                         *link;
```

Comment:
Was this link pointer never used?

Reply:
This link pointer was used but I realized my stupidity and removed it.